sdk/communication/azure-communication-chat/src/main/java/com/azure/communication/chat/models/ChatRetentionPolicy.java

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.communication.chat.models;
+
+import com.azure.core.annotation.Immutable;
+import com.azure.json.JsonReader;
+import com.azure.json.JsonSerializable;
+import com.azure.json.JsonToken;
+import com.azure.json.JsonWriter;
+
+import java.io.IOException;
+
+/**
+ * Data retention policy for auto deletion.
+ */
+@Immutable
+public class ChatRetentionPolicy implements JsonSerializable<ChatRetentionPolicy> {
+    /*
+     * Retention Policy Type
+     */
+    private com.azure.communication.chat.models.RetentionPolicyKind kind
+        = com.azure.communication.chat.models.RetentionPolicyKind.fromString("ChatRetentionPolicy");
+
+    /**
+     * Creates an instance of ChatRetentionPolicy class.
+     */
+    public ChatRetentionPolicy() {
+    }
+
+    /**
+     * Get the kind property: Retention Policy Type.
+     *
+     * @return the kind value.
+     */
+    public com.azure.communication.chat.models.RetentionPolicyKind getKind() {
```

Comment:
You don't need the fully qualified name of the public `RetentionPolicyKind` in the return Type if you're not using the impl version anywhere in the class.